clients/ui/bff/internal/data/model_registry.go

```diff
@@ -18,7 +18,7 @@ func (m ModelRegistryModel) FetchAllModelRegistries(client k8s.KubernetesClientI
 		return nil, fmt.Errorf("error fetching model registries: %w", err)
 	}
 
-	var registries []ModelRegistryModel
+	var registries []ModelRegistryModel = []ModelRegistryModel{}
```

Comment:
@alexcreasy @ederign And I did this just to not get a null when unmarhsalling the value.

Reply:
Another way to avoid this is to create a slice with the size of resources! But if this works we are good to go.